x-pack/plugins/ml/public/application/jobs/jobs_list/components/jobs_list/jobs_list.js

```diff
@@ -71,7 +71,7 @@ export class JobsList extends Component {
       return id;
     }
 
-    return <ADJobIdLink id={id} />;
+    return <ADJobIdLink key={id} id={id} />;
```

Comment:
Nit: Besides `Ml` as a prefix we hardly use any abbreviations in component names so I was wondering if we should make this the full written `<AnomalyDetectionJobIdLink ... />`.

Reply:
Updated here cc8f5dd